sdk/tables/azure-data-tables/migration_guide.md

```diff
@@ -0,0 +1,298 @@
+# Guide for migrating to `azure-data-tables` from `azure-cosmosdb-table`
+
+This guide is intended to assist in the migration to `azure-data-tables` from `azure-cosmosdb-table`. It will focus on side-by-side comparisons for similar operations between the two packages.
+
+We assume that you are familiar with `azure-cosmosdb-table`. If not, please refer to the README for [`azure-cosmosdb-table`](https://github.com/Azure/azure-cosmos-table-python/tree/master/azure-cosmosdb-table) rather than this guide.
```

Comment:
Why would we be pointing them to the old repo? Wouldn't we be better of specifying the target audience as being people who have code built around `azure-cosmosdb-table` - and for those that do not, simply direct them to `azure-data-tables`?